backend/cellguide/pipeline/computational_marker_genes/__init__.py

```diff
@@ -61,6 +61,29 @@ def get_computational_marker_genes(*, snapshot: WmgSnapshot, ontology_tree: Onto
         else:
             marker_genes[key] = marker_genes_per_tissue[key]
 
+    # convert all groupby_dims IDs to labels as required by CellGuide
```

Comment:
This step was brought out of the MarkerGeneCalculator pipeline as WMG needs IDs, not labels. This way MarkerGeneCalculator can be used for both cellguide and wmg.